src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -7255,6 +7255,19 @@ void M($$Person p)
 }", MainDescription("record struct Person"));
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public async Task QuickInfoReadOnlyRecordStruct()
+        {
+            await TestWithOptionsAsync(
+                Options.Regular.WithLanguageVersion(LanguageVersion.CSharp9),
```

Comment:
I just noticed I unintentionally left the language version from copy/pasting the test above. But the test should probably fail with C# 9.0

However, It's passing. Feels weird.

Reply:
That's expected. It should pass in C# 9. With some rare exceptions, we always bind in the latest version of the language, only producing different diagnostics. So the symbol is the same regardless of language version.